CMakeLists.txt

```diff
@@ -1,39 +1,61 @@
 cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
+
+include(gen/CurrentBuildVulkanVersion.cmake)
+
 project(VulkanBootstrap
     LANGUAGES CXX
-    DESCRIPTION "A Vulkan utility library to ease the initialization steps in Vulkan")
+    DESCRIPTION "A Vulkan utility library to ease the initialization steps in Vulkan"
+    VERSION ${VK_BOOTSTRAP_SOURCE_HEADER_VERSION})
```

Comment:
I really don't like setting the version like this.

I'd prefer the version be explicit here and make the testing verify the project version is correct.

It's a bit more work but I think it's more elegant solution overall.

Reply:
How do you set the project version manually? Is it per target or per project?

Reply:
```
VERSION 1.3.272)
```

Then you test the project version in your test directory.

```
if (NOT PROJECT_VERSION VERSION_EQUAL VK_BOOTSTRAP_SOURCE_HEADER_VERSION)
    message(FATAL_ERROR "Update project version")
endif()
```

Reply:
But how do I update that automatically? Does the update script modify the CMakelists.txt inplace?

Reply:
Looking at `Vulkan-Headers`, I see that it uses `${VK_VERSION_STRING}` in the project().